File: 240531/함수를 이용한 온전수 판별/determining-the-whole-number-using-a-function.py
def is_check(num):
    if num % 2 == 0:
        return False
    
    if num % 10 == 5:
        return False
    
    if num % 3 == 0 and num % 9 != 0:
        return False
    
    return True

a, b = tuple(map(int, input().split()))

cnt = 0

for num in range(a, b+1):
    if is_check(num):
        cnt += 1

print(cnt)

File: 241230/함수를 이용한 온전수 판별/determining-the-whole-number-using-a-function.cpp
#include <iostream>
using namespace std;

bool check (int num) {
    if (num%2 == 0) {
        return false;
    }
    if (num%10 == 5) {
        return false;
    }
    if (num%3 == 0 && num%9 != 0) {
        return false;
    }
    return true;
}

int main() {
    // 여기에 코드를 작성해주세요.
    int a, b, cnt=0;
    cin >> a >> b;
    for (int i=a; i<=b; i++) {
        if (check(i)) cnt++;
    }
    cout << cnt;
    return 0;
}